rpa-builder/6.4/modules/ROOT/pages/toolbox-excel-operations-excel-session.adoc

```diff
@@ -108,39 +98,23 @@ This Action Step has no outbound variables.
 
 == Wizard
 
-In the *Operation Mode* section, first choose whether you want to only read the Excel file, read and
-modify it or read it and write to a different Excel file. If the _Read
-Only_ mode is selected, the source file is read only and cannot be modified.
+Use the *Operation Mode* section to choose whether to only read the Excel file, read and modify, or read and then write to a different Excel file. 
 
-The *source file* is the file that should be read. You can write the path to the directory where the source file
-is located directly in the input field, insert Environment and Script Variables, make a selection using a file explorer
-//image:media\image1.png[image,width=20,height=20]
-or connect with a Pin Variable. You also have the option of using a different file at the time
-of execution to at the time of creation.
-//Simply select the checkbox image:media\image2.png[image,width=303,height=15].
+The *Source file* is the file to read. Specify the path to the directory where the source file
+is located directly in the input field, inserting Environment and Script Variables, selecting it using a file explorer, or by using a Pin Variable. 

```

Comment:
This sentence doesn't really make sense to me. Do you specify the directory by inserting the environment?